loleaflet/src/map/handler/Map.WOPI.js

```diff
@@ -116,6 +117,8 @@ L.Map.WOPI = L.Handler.extend({
 		this.SupportsRename = !!wopiInfo['SupportsRename'];
 		this.UserCanRename = !!wopiInfo['UserCanRename'];
 		this.EnableShare = !!wopiInfo['EnableShare'];
+		this.IsFreemiumUser = !!wopiInfo['IsFreemiumUser'];
+		this._map._freemiumBlacklist = wopiInfo['FreemiumBlacklist'];
```

Comment:
As mentioned elsewhere, wopiInfo is for WOPI. Let's not abuse it please. Maintenance becomes harder when we have unrelated things together. If we need to pass some info to the client, there are already well-established ways of doing that. Feel free to ask for pointers, but there are many examples already.

Reply:
updated